.travis.yml

```diff
@@ -18,6 +18,8 @@ matrix:
   ## linters for json and JavaScript files
     - language: node_js
       python:
+      node_js:
+        - 7
```

Comment:
Why not 8?  Current node_js is v8.8.0.  (I am not a JavaScript guy so forgive me if the answer is self-evident.)